modules/dists/CyclicDist.chpl

```diff
@@ -759,7 +764,7 @@ proc CyclicDom.dsiDims() do        return whole.dims();
 proc CyclicDom.dsiGetIndices() do  return whole.getIndices();
 proc CyclicDom.dsiMember(i) do     return whole.contains(i);
 proc CyclicDom.doiToString() do    return whole:string;
-//proc CyclicDom.dsiSerialWrite(x) { x.write(whole); }
+//proc CyclicDom.dsiSerialWrite(x) throws { x.write(whole); }
```

Comment:
I'm assuming this was updated via a script instead of due to actually having gotten a warning as a result of it - are we sure this is necessary?  I don't think it's necessarily worth backing out, it just seemed funny to me :)

Reply:
Not a script, just figuring that if/when someone went to uncomment this, they'd need the `throws`, so might as well keep the comment current.  I'm sure the `throws` is necessary if this routine is used, but I didn't look into why it's commented out to begin with.

Reply:
Yeah, I suspect it will also be helpful if it gets uncommented as well, assuming it doesn't just conflict with the version you did update

Reply:
Ah—you looked at the code more carefully than I did!  :D  Probably better would've been to just remove the line, but I agree with your original assessment that it doesn't seem worth worrying about if you stand by that.

Reply:
I might be tempted to add a comment that it might be unnecessary/handled later in the file?  But other than that, I don't think it's worth worrying about too much